src/components/Episodes/Episodes.jsx

```diff
@@ -8,23 +9,20 @@ import {
 
 export default function Episodes(){
 
-	const pagination = data.map(RENDER.pagination);
 	const episodes = data.map(RENDER.episode);
 
 	return (
-		<article>
-			<EpisodesProvider>
-				<ol>
-					{ episodes }
-				</ol>
-				<nav>
-					<button aria-label="Previous episode." />
-					<button aria-label="Next episode." />
+		<>
+			<article>
+				<EpisodesProvider>
 					<ol>
-						{ pagination }
+						{ episodes }
 					</ol>
-				</nav>
+				</EpisodesProvider>
+			</article>
+			<EpisodesProvider>
+				<EpisodesControls />
```

Comment:
SUPER minor nitpick - could we name this component `<EpisodesNav />` instead? 🙏

Reply:
Yarp. done